src/ray/raylet/node_manager.cc

```diff
@@ -662,7 +665,7 @@ void NodeManager::FinishAssignedTask(Worker &worker) {
     auto actor_notification = std::make_shared<ActorTableDataT>();
     actor_notification->actor_id = actor_id.binary();
     actor_notification->actor_creation_dummy_object_id =
-        task.GetTaskSpecification().ActorCreationDummyObjectId().binary();
+        task.GetTaskSpecification().ActorDummyObject().binary();
```

Comment:
Wasn't this correct before?

Reply:
`ActorCreationDummyObjectId` is only set for actor methods, not for the actor creation task. :(